WtWSMS/common/landed_titles/BP_e_byzantium.txt

```diff
@@ -12039,31 +13369,120 @@ e_byzantium = {
 			c_aytos = {
 				color = { 65 80 55 }
 				cultural_names = {
+					#Roman Cultures
+					name_list_roman = cn_aquilia
+					name_list_romano_gallic = cn_aquilia
+					name_list_romano_hispanic = cn_aquilia
+					name_list_romano_british = cn_aquilia
+					name_list_romano_african = cn_aquilia
+					name_list_romano_raetian = cn_aquilia
+					name_list_romano_thracian = cn_aquilia
+					name_list_romano_illyrian = cn_aquilia
+					name_list_romano_pannonian = cn_aquilia
+					name_list_dalmatian = cn_aquilia
+					name_list_romano_noric = cn_aquilia
+					name_list_romano_gothic = cn_aquilia
+					name_list_romano_dacian = cn_aquilia
+					name_list_romano_frank = cn_aquilia
+					name_list_romano_aramean = cn_aquilia
+					name_list_romano_vandalic = cn_aquilia
 					#Byzantine Cultures
 					name_list_greek = cn_aetos
+					name_list_aeolian = cn_aetos
+					name_list_helleno_aramean = cn_aetos
+					name_list_greek_gothic = cn_aetos
+					name_list_greek_persian = cn_aetos
+					name_list_isaurian_persian = cn_aetos
+					name_list_grecco_axumite = cn_aetos
+					name_list_bosporan = cn_aetos
+					name_list_pontic = cn_aetos
+					name_list_tsakonian = cn_aetos
+					name_list_massalian = cn_aetos
+					name_list_helleno_thracian = cn_aetos
+					name_list_helleno_libyan = cn_aetos
+					#Proto-Carpathian Cultures
+					name_list_thracian = cn_aetos
 					#Turkic Cultures
-					name_list_turkish = cn_devne
-					#South Slavic Cultures
-					name_list_bulgarian = cn_devnya
+					name_list_turkish = cn_aydos
```

Comment:
Why are you removing Bulgarian?

Reply:
> Why are you removing Bulgarian?

Because the default name, Aytos, is Bulgarian,

Reply:
Yes, but wasn't Devyna a far more important settlement here in the period?

Reply:
> Yes, but wasn't Devyna a far more important settlement here in the period?

Yes, but it is 94 km away.